Core/src/main/java/com/plotsquared/core/util/StringMan.java

```diff
@@ -320,4 +323,14 @@ public static boolean contains(String name, char c) {
         return col;
     }
 
+    public static final Pattern ALLOWED_PATTERN = Pattern.compile("(?<quoted>\"[\\w ]+\")|(?<single>\\w+)");
```

Comment:
This should be private, and the name is barely telling anything about the functionality.

Reply:
is private now.